test/internet/test-dns-any.js

```diff
@@ -59,9 +59,6 @@ const checkers = {
   checkTXT(r) {
     assert.ok(Array.isArray(r.entries));
     assert.ok(r.entries.length > 0);
-    r.entries.forEach((txt) => {
```

Comment:
What about something like:

```
assert(r.entries.some((txt) => txt.startsWith('v=spf1')));
```

Reply:
The problem is the records will be in different 'r's, instead of different entries in the same 'r' (probably something we need to fix, though should be in another PR IMO